src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Checkbox, Col } from '@folio/stripes/components';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useDerivativeModification } from '../../../../../hooks';
+import { ACTION_PARAMETERS_KEY } from './helpers';
+import css from '../BulkEditInApp.css';
+
+export const AdditionalActionParameters = ({ action, actionIndex, onChange }) => {
+  const { formatMessage } = useIntl();
+  const dynamicWidth = action.parameters * 2;
```

Comment:
If `action` prop is not required, then this should be `action?.parameters * 2`. Please check prop types and how other props are accessed

Reply:
thanks, fixed